software/firmware/calibrate.py

```diff
@@ -106,10 +106,13 @@ def wait_for_b1(value):
         cv1.duty_u16(duty)
         duty += 10
         reading = sample()
+        #print('duty: ' + str(duty) + ' expected: ' + str(expected_reading) + ' got: ' + str(reading))
+    print(str(index+1) + 'V duty: ' + str(duty))
```

Comment:
The calibrate script is intended to be headless, so we'd like to keep the print statements out. Was there a problem this was trying to solve? If so, lets fix that problem or open up a ticket to tackle later.